pkg/gds/gds_test.go

```diff
@@ -52,82 +74,136 @@ func loadFixtures(s *gdsTestSuite) {
 		// Unmarshal the JSON into the global fixtures map.
 		data, err := os.ReadFile(path)
 		require.NoError(err)
-		if strings.HasPrefix(info.Name(), "vasps::") {
+		key := strings.TrimSuffix(info.Name(), ".json")
+		switch {
+		case strings.HasPrefix(key, vaspPrefix):
 			vasp := &pb.VASP{}
 			err = protojson.Unmarshal(data, vasp)
 			require.NoError(err)
-			dbVASPs[info.Name()] = vasp
+			s.fixtures[key] = vasp
 			return nil
-		}
-		if strings.HasPrefix(info.Name(), "certreqs::") {
+		case strings.HasPrefix(key, certPrefix):
 			cert := &models.CertificateRequest{}
 			err = protojson.Unmarshal(data, cert)
 			require.NoError(err)
-			dbCerts[info.Name()] = cert
+			s.fixtures[key] = cert
 			return nil
 		}
 		return fmt.Errorf("unrecognized prefix for file: %s", info.Name())
 	})
 	require.NoError(err)
-	os.RemoveAll(root)
-	require.NoError(err)
+	require.NoError(os.RemoveAll(root))
+}
+
+// writeObj writes a protobuf object to the given database.
+func (s *gdsTestSuite) writeObj(db *leveldb.DB, key string, obj interface{}) {
```

Comment:
@rebeccabilbro and I encountered an issue in the trtl test fixtures, where by circumventing the database insertion mechanisms, our fixtures and expected results did not match. To prevent that from happening to us here, I'm going to update this to directly use the store from the service.